packages/SwingSet/tools/prepare-test-env.js

```diff
@@ -6,9 +6,14 @@
  * for virtual objects: makeKind, makeWeakStore
  */
 
+import { wrapTest } from '@agoric/ses-ava';
 import '@agoric/install-ses';
+import rawTest from 'ava';
+
 import { makeFakeVirtualObjectManager } from './fakeVirtualObjectManager';
 
+const test = wrapTest(rawTest);
```

Comment:
You're not exporting this `test`

Reply:
I'm not intending to. This is imported directly without importing an export.